doc/users/next_whats_new/legend_title_prop_kwarg.rst

```diff
@@ -0,0 +1,9 @@
+Legend now has a `title_prop` kwarg
+-------------------------------------
+
+The title for a `.Figure.legend` and `.Axes.legend` can now have its
+title's font properties set via the ``title_prop`` kwarg, defaults to 
+``None``, which means the legend's title will have the font properties
+ as set by the :rc:`legend.title_fontsize`. Through this new kwarg, one
+can set the legend's title font properties through either FontProperties or 
+dict (setting 'family','style','variant','size' and 'weight'). 
```

Comment:
rather than abstract, can you just put an example of the dict in here?

Reply:
Thanks. I will add an example of the dict.